ckan/cli/search_index.py

```diff
@@ -77,6 +77,43 @@ def clear(dataset_name: str):
         clear_all()
 
 
+def list_orphans(return_list: bool=False) -> list|None:
+    import ckan.logic as logic
+    search = logic.get_action('package_search')({},{
```

Comment:
I believe that `package_search` returns 10-20 records by default. What if there  are more of them?

Reply:
There is now a loop for `package_search` at an iteration of 1000 which was described as the "upper limit" in `ckan/logic/action/get/package_search`. Though, this particular `package_search` is only getting the `id` fields so should be fine by the 1000.